test/e2e/common/pod_resize.go

```diff
@@ -49,7 +49,11 @@ const (
 	CgroupMemLimit  string = "/sys/fs/cgroup/memory/memory.limit_in_bytes"
 
 	PollInterval time.Duration = 2 * time.Second
-	PollTimeout  time.Duration = time.Minute
+	// PollTimeout  time.Duration = time.Minute
+	PollTimeout  time.Duration = 5 * time.Minute
+
+	// Bustable containers without CPU request will have a default "2m" cpu request
+	NotSpecifiedCPUReq = "2m"
```

Comment:
Suggest renaming it to DefaultMinCpuRequests

Reply:
Sure, will do.